src/components/Quote.js

```diff
@@ -1,4 +1,5 @@
 import React, { useState, useEffect } from 'react';
+import './styles.css';
 
 function DisplayQuote() {
   const [data, setData] = useState([{ quote: '', author: '', category: '' }]);
```

Comment:
- [ ] Good job with this project, you have really shown great care and commitment to clean code practice :clap: 
However, even though the style implementation is not too rigid, there are still some basic requirements to be met. For example, this quote page indicates that the `quote` should be aligned to the center of the page for a good user experience and as a matter of accessible user interface rather than shifting to an extreme angle when we have excessive space wasting on the opposite side :hammer_and_wrench: 

![image](https://github.com/yusufyusie/math-magicians/assets/99715640/57bc1019-d3d1-4f51-bf92-d5ca2e2981d9)